lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1074,6 +1074,15 @@ LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
       /*out=*/remPipe.out_remainder());
 }
 
+namespace {
+
+// https://github.com/llvm/circt/issues/2660
+inline unsigned handleZeroWidth(int64_t dim) {
```

Comment:
I think inside an anonymous namespace you don't need inline, or you could remove the anonymous an use static, at least, that is how I'm reading https://llvm.org/docs/CodingStandards.html#anonymous-namespaces.

Reply:
Honestly, I'm not sure the recommendation for when to use `inline`, so copying upstream here sounds good.

Reply:
changed to `static` (since, as alluded to in llvm link, it's equivalent in this case of the single function) and then remembered @cgyurgyik's ask for tests. Not sure if the test is *exactly* the right pattern (just `CHECK-NEXT` all the things) but it does pass.